scripts/common

```diff
@@ -17,7 +17,7 @@ readonly SERVER_DOCKER_IMAGE_NAME='gcr.io/oak-ci/oak-server'
 readonly CACHE_DIR='bazel-cache'
 
 # List all workspaces, and standalone crates.
```

Comment:
Are there any standalone crates left?

Reply:
Edit: yes, I think so &ndash; `expect` and `roughenough`. 

- __`./Cargo.toml`:__
    - `./oak/server/rust/oak_loader/Cargo.toml`
    - `./oak/server/rust/oak_runtime/Cargo.toml`
- __`./examples/Cargo.toml`:__
    - `./examples/abitest/abitest_common/Cargo.toml`
    - `./examples/abitest/module_0/rust/Cargo.toml`
    - `./examples/abitest/module_1/rust/Cargo.toml`
    - `./examples/abitest/tests/Cargo.toml`
    - `./examples/aggregator/backend/Cargo.toml`
    - `./examples/aggregator/common/Cargo.toml`
    - `./examples/aggregator/module/rust/Cargo.toml`
    - `./examples/authentication/client/Cargo.toml`
    - `./examples/chat/module/rust/Cargo.toml`
    - `./examples/hello_world/module/rust/Cargo.toml`
    - `./examples/machine_learning/module/rust/Cargo.toml`
    - `./examples/private_set_intersection/module/rust/Cargo.toml`
    - `./examples/running_average/module/rust/Cargo.toml`
    - `./examples/translator/common/Cargo.toml`
    - `./examples/translator/module/rust/Cargo.toml`
    - `./examples/trusted_information_retrieval/client/rust/Cargo.toml`
    - `./examples/trusted_information_retrieval/module/rust/Cargo.toml`
- __`./experimental/Cargo.toml`:__
    - `./experimental/split_grpc/client/Cargo.toml`
    - `./experimental/split_grpc/proxy/Cargo.toml`
    - `./experimental/split_grpc/server/Cargo.toml`
- __`./oak_abi/Cargo.toml`__
- __`./oak_utils/Cargo.toml`__
- __`./runner/Cargo.toml`__
- __`./sdk/Cargo.toml`:__
    - `./sdk/rust/oak/Cargo.toml`
    - `./sdk/rust/oak_tests/Cargo.toml`
- `./third_party/expect/Cargo.toml`
- `./third_party/roughenough/Cargo.toml`